Command/ComposerUpdateCommand.php

```diff
@@ -32,16 +45,16 @@ protected function configure(): void
             ->setHelp('This command allows you to run further installation an configuration actions afther installing a plugin');
     }
 
+    /**
+     * @param InputInterface  $input  Symfony style
+     * @param OutputInterface $output Symfony style
+     *
+     * @return int Succes or failure of the command
+     */
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
         $this->installationService->setStyle(new SymfonyStyle($input, $output));
 
-        $bundle = $input->getOption('bundle');
```

Comment:
Where do we get and use these options?
Because they are configured for this command,
But there is no code present in this branch that gets and uses these options, as far as i can see.